app/modules/ads/ad-context.js

```diff
@@ -5,6 +5,7 @@ export default {
 	bidders: {
 		enabled: false,
 		timeout: 2000,
+		disabledSlots: [],
```

Comment:
https://github.com/Wikia/ad-products/pull/131/files#r209212784 :(

Reply:
It stays empty on MW+AE3, right?

Reply:
Yep, it's not necessary.